pkgs/tools/package-management/nix-doc/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, rustPlatform, fetchFromGitHub, pkgs, ... }:
```

You shouldn't use `pkgs` in derivations, rather import the required packages directly.

```suggestion
{ stdenv, rustPlatform, fetchFromGitHub, pkg-config, boost, nix }:
```